lib/ansible/modules/network/aci/aci_rest.py

```diff
@@ -240,7 +237,7 @@ def main():
     aci = ACIModule(module)
 
     if method == 'get':
-        aci.request()
+        aci.request(path)
```

Comment:
Pretty hard if we don't have the gear or a software implementation ;-)

Please look at: https://github.com/ansible/community/wiki/Network:-ACI to understand where we are, where we would like to go and what is blocking us :-)